src/quiz.js

```diff
@@ -2,75 +2,89 @@ import QuizUI from "./ui/QuizUI";
 import WordService from "./service/WordService";
 import Util from "./lib/Util";
 import Progress from "./model/Progress";
-import {LOCALSTORAGE_KEY} from './Constant';
+import { CURR_USER_KEY } from "./Constant";
+import EasyHttp from "./lib/EasyHttp";
```

Comment:
You should not need to directly import `EasyHttp`. An alternative suggestion would be to use one of the services instead, like `UserService`. Only services should have contact with `EasyHttp`.